### PR TITLE
Add postgresql10 test

### PR DESCRIPTION
**Pull request by dzedro**
- Related ticket: https://progress.opensuse.org/issues/46877
- Verification run:
[12SP2](http://10.100.12.155/tests/10329#step/php7_postgresql/66)
[12SP3](http://10.100.12.155/tests/10334#step/php7_postgresql/66)
[12SP4](http://10.100.12.155/tests/10336#step/php7_postgresql/66)
[15](http://10.100.12.155/tests/10330#step/php7_postgresql/66)

### CODE REVIEW

File: tests/console/postgresql10.pm [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: postgresql10 regression test
```
[the remark this answers is not in the document]
Comment (dzedro) [reply tests/console/postgresql10.pm]: spoke with @mimi1vx , I will try to merge this into postgresql_server.pm

**Comment by dzedro** [comment]
merged with existing test_pgsql()

**Comment by okurz** [comment]
+1

how about an openSUSE test?

**Comment by dzedro** [comment]
http://10.100.12.155/tests/10340#step/php7_postgresql/66

opensuse does even have version 93 which is not expected in the code, but it doesn't really matter

**Pull request merged**